andela/bp-esa-backend
# #164396754 Ft Add Pagination

#### What does this PR do?

- Returns pagination data

#### Description of Task to be completed?
- This will provide a smaller chunk of a report to the user, help him/her to read the report page easier and also include parameter based filtering.

#### How should this be manually tested?
1. Clone the project and run it
2. On an app like Postman, send url in this format `/api/v1/automations?page={int}&limit={int}`; 
**limit** being the number of items you expect the pagination to return, and **page** being the page number: **### example below**

![3lwmpgdfsb](https://user-images.githubusercontent.com/25157392/53883527-4744b500-402a-11e9-8c7f-759c0f75cd2d.gif)

#### What are the relevant pivotal tracker stories?

[#164396754](https://www.pivotaltracker.com/n/projects/2197045/stories/164396754)
